modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONOrdersAPI.macro

```diff
@@ -26,6 +26,14 @@ definition {
 			var deliveryTermId = CommerceJSONTermsAndConditionsAPI._getCommerceTermsAndConditionsIdByName(termsAndConditionsName = ${deliveryTermName});
 		}
 
+		if (!(isSet(orderDate))) {
+			var displayDateYear = DateUtil.getCurrentYear();
+
+			var displayDateYear = ${displayDateYear} + 1;
+
+			var orderDate = "${displayDateYear}-01-01T01:01:00Z";
+		}
+
```

Comment:
@Brunno0liveira this is causing functional tests to fail, please change this so that only if 'orderDate' is set, then the field will be added to the payload